lib/active_yaml/aliases.rb

```diff
@@ -5,22 +5,31 @@ def self.included(base)
       base.extend(ClassMethods)
     end
 
+    ALIAS_KEY_REGEXP = /^\//i
+
     module ClassMethods
 
       def insert(record)
         super if record.attributes.present?
       end
 
       def raw_data
-        super.reject do |k, v|
-          v.kind_of? Hash and k.match(/^\//i)
+        d = super
+        if d.is_a?(Array)
```

Comment:
Will `kind_of?` still work?

```suggestion
        if d.kind_of?(Array)
```

Reply:
Since `kind_of?` is an alias of `is_a?`, it works.
https://docs.ruby-lang.org/en/3.0.0/Object.html#method-i-kind_of-3F